### PR TITLE
update takeoff config

### PR DESCRIPTION
**Pull request by timvancann**
## Checklist:
  - [ ] The code change is tested and works locally.
  - [ ] There is no commented out code in this PR.
  - [ ] There are no new TODOs in this PR

If exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated

### CODE REVIEW

File: docs/takeoff_config.md [hunk 1]
```diff
@@ -6,36 +6,175 @@ permalink: takeoff-config
 ---
 
 # Overview
-The `takeoff_config.yaml` is the place put any variables that should not be hardcoded in python. When [developing a new component for Takeoff](deveploment_guide) you can use these variables in your component authenticate against Azure services using the Azure KeyVault as central place to store these credentials.
+The `.takeoff/config.yaml` is the place put any variables that should not be hardcoded in python. When you can use these variables in your component authenticate against Cloud services.
```
Comment (danielvdende) [inline docs/takeoff_config.md]: The second sentence doesn't make much sense :sweat_smile:

File: docs/takeoff_config.md [hunk 2]
```diff
@@ -6,36 +6,175 @@ permalink: takeoff-config
...
+  databricks_fs_libraries_mount_path: "dbfs:/mnt/libraries"
 ```
 
 {:.table}
 | field | description |
 | ----- | ----------- |
-| `resource_group` | A python [format string](https://docs.python.org/3/library/stdtypes.html#str.format) capturing the naming conventions of the Azure resource groups across `DAP`
-| `location` |  [Azure location](https://azure.microsoft.com/en-us/global-infrastructure/locations/) of the Data center 
-| `vault_name` | A python [format string](https://docs.python.org/3/library/stdtypes.html#str.format) capturing the naming conventions of the Azure KeyVault across `DAP`
+| `databricks_fs_libraries_mount_path` | Path on [`dbfs`](https://docs.databricks.com/user-guide/databricks-file-system.html) where libraries (such as wheels and jars) are stored. Usually this is a mounted directory to cloud storage.
```
Comment (danielvdende) [inline docs/takeoff_config.md]: `Usually this is a mounted directory to cloud storage` -> `Usually this is a mounted cloud storage path` ?

Comment (timvancann) [reply docs/takeoff_config.md]: Better!

File: docs/takeoff_config.md (hunk 1 above)
Comment (timvancann) [reply docs/takeoff_config.md]: Fixed. It didn't :p

**Pull request merged**